src/Symfony/Component/Messenger/Tests/Transport/Serialization/SerializerTest.php

```diff
@@ -61,4 +80,26 @@ public function testUsesTheCustomFormatAndContext()
         $this->assertSame('Yay', $encoded['body']);
         $this->assertSame($message, $decoded);
     }
+
+    public function testEncodedWithSerializationConfiguration()
+    {
+        $serializer = new Serializer(
+            new SerializerComponent\Serializer(array(new ObjectNormalizer()), array('json' => new JsonEncoder()))
+        );
+
+        $message = new ConfiguredMessage(new DummyMessage('Hello'));
+        $message
+            ->with(new SerializerConfiguration(array(ObjectNormalizer::GROUPS => array('foo'))))
+            ->with(new ValidationConfiguration(array('foo', 'bar')))
```

Comment:
Is it possible add another test for case when validation groups provided as `GroupSequence` object?

Reply:
That would be in dedicated unit tests ensuring the configurations are properly serializable :)

Reply:
Done